Tests/SpeziFHIRTests/FHIRStore/FHIRStoreObservationChangesTests.swift

```diff
@@ -0,0 +1,260 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import ModelsR4
+@testable import SpeziFHIR
+import XCTest
+
+
+final class FHIRStoreObservationChangesTests: XCTestCase {
+    private var store: FHIRStore! // swiftlint:disable:this implicitly_unwrapped_optional
```

Comment:
We might just do something like that:
```
private var store: FHIRStore = FHIRStore()
```

Barley costs anything and avoid the force unwrap.